lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Dish
+
+
+attr_accessor :name, :price
+
+  def initialize name, price=1
+    @name = name
+    @price = price
+  end
+
```

Comment:
Use attr_reader here instead of attr_accessor. Better to write individual setter methods so that you can control the input to ensure that the name is always a string, the price is always an integer, and any malicious or faulty code in other classes cannot delete your data.